File: Typescript/test/isBalanced.test.ts
import { expect, test } from '@jest/globals';
import * as solution from '../src/isBalanced';

test('Example 1', () => {
	let input = "1234";
	let expected = false;
	expect(solution.isBalanced(input)).toEqual(expected);
});

test('Example 2', () => {
	let input = "24123";
	let expected = true;
	expect(solution.isBalanced(input)).toEqual(expected);
});

File: Dart/test/isBalanced_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:leetcode/isBalanced.dart';

void main() {
  test("test example 1", () {
    var input = "1234";
    var expected = false;

    final sln = Solution();
    var output = sln.isBalanced(input);
    expect(output, expected);
  });
  test("test example 2", () {
    var input = "24123";
    var expected = true;

    final sln = Solution();
    var output = sln.isBalanced(input);
    expect(output, expected);
  });
}
